samples/client/src/main/java/global/covesa/sdk/client/push/MockApplicationServer.kt

```diff
@@ -0,0 +1,221 @@
+package global.covesa.sdk.client.push
+
+import android.content.Context
+import android.security.keystore.KeyGenParameterSpec
+import android.security.keystore.KeyProperties
+import android.util.Log
+import com.android.volley.RequestQueue
+import com.android.volley.Response
+import com.android.volley.VolleyError
+import com.android.volley.toolbox.StringRequest
+import com.android.volley.toolbox.Volley
+import com.google.crypto.tink.apps.webpush.WebPushHybridEncrypt
+import global.covesa.sdk.api.client.push.data.PushEndpoint
+import org.json.JSONObject
+import java.net.URL
+import java.security.KeyPair
+import java.security.KeyPairGenerator
+import java.security.KeyStore
+import java.security.KeyStore.PrivateKeyEntry
+import java.security.Signature
+import java.security.interfaces.ECPublicKey
+import java.security.spec.ECGenParameterSpec
+
+/**
+ * This class emulates an application server, this should not be implemented
+ * by real applications.
+ */
+class MockApplicationServer(private val context: Context) {
```

Comment:
In Android, Mocks usually refer to class with stub methods which do not do anything, I would tend to call this class a `FakeApplicationServer` instead.